docs/usage.rst

```diff
@@ -0,0 +1,103 @@
+.. include:: links.rst
+
+How to Use
+============
```

Comment:
Title lines should match the length of the title.

```suggestion
==========
```

This is being captured by the github action:

```
Warning, treated as error:
/home/runner/work/eddymotion/eddymotion/docs/usage.rst:7:Title underline too short.
```

Before merging the build docs action (https://github.com/nipreps/eddymotion/actions/runs/8718955800/job/23917263784?pr=169) must be green.